src/components/data-table-v2/_data-table-v2-sort.scss

```diff
@@ -66,3 +66,118 @@
     min-width: 14px;
   }
 }
+
+@mixin data-table-v2-sort--x {
+  // -------------------------------------
+  // Sortable table
+  // -------------------------------------
+
+  .#{$prefix}--data-table-v2--sort th:hover {
+    background: $data-table-column-hover;
+    border-top: 1px solid $data-table-column-hover;
+    border-bottom: 1px solid $data-table-column-hover;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-icon:hover,
```

Comment:
What are these classes being used for on the `th`s?

Reply:
These are being used to target `th`'s that aren't actually sortable/actionable from the table header (like checkbox). Design does not want these to have a background color change on hover.

Reply:
Could we instead describe the styles as opt-in so the default would be no hover? One idea could be to target `bx--table-sort-v2` for when to add hover and focus styles for sortable headers.

Reply:
Is this simply to not have a default hover when it's sortable? or is this because of the selectors? Some of these selectors are used for other things in different PRs so not using them here wouldn't change the need for them (i.e. different padding https://github.com/IBM/carbon-components/pull/2121#discussion_r267559960, and expandable styles https://github.com/IBM/carbon-components/blob/d8c014584240035b809848f07c36db54e119ad30/src/components/data-table-v2/_data-table-v2-expandable.scss#L78-L85)

Reply:
@jnm2377 it's because we are adding additional classes and I'm just to brainstorm what will be the minimum cost to change here 👍 That's all.